sgqlc/endpoint/websocket.py

```diff
@@ -65,7 +69,10 @@ def __call__(self, query, variables=None, operation_name=None):
                                          **self.ws_options)
         try:
             init_id = self.generate_id()
-            ws.send(json.dumps({'type': 'connection_init', 'id': init_id}))
+            connection_setup_dict = {'type': 'connection_init', 'id': init_id}
+            if self.connection_params:
+                connection_setup_dict['payload'] = self.connection_params
```

Comment:
given this, I'd rather call `connection_payload`

Reply:
Yep, no problem.